src/agoradatatools/etl/transform/gene_info.py

```diff
@@ -15,9 +15,9 @@ def transform_gene_info(
     igap = datasets["igap"]
     eqtl = datasets["eqtl"]
     proteomics = datasets["proteomics"]
-    rna_change = datasets["rna_expression_change"]
-    proteomics_tmt = datasets["agora_proteomics_tmt"]
-    proteomics_srm = datasets["agora_proteomics_srm"]
+    rna_change = datasets["diff_exp_data"]
```

Comment:
Just wanted to make sure this doesn't change the result in any way.

Reply:
I did not notice any differences while testing the changes, but maybe @jaclynbeck-sage or @JessterB can confirm.

Reply:
I've just confirmed that the `gene_info.json` file created by CI testing for this PR (https://www.synapse.org/#!Synapse:syn17015359 version 216) is identical to my local copy of `gene_info.json`, which was created from the dev branch.

Reply:
Thank you @jaclynbeck-sage!